packages/front-end/components/Experiment/TabbedPage/ProjectTagBar.tsx

```diff
@@ -36,111 +35,124 @@ export default function ProjectTagBar({
+              >
+                {projectId && <strong>{projectName}</strong>}{" "}
+                <FaExclamationTriangle className="text-warning" />
+              </Tooltip>
+            ) : (
+              projectId && <strong>{projectName}</strong>
+            )}
+            {editProject && !projectId && (
+              <a
+                role="button"
+                className="cursor-pointer button-link"
+                onClick={(e) => {
+                  e.preventDefault();
+                  editProject();
+                }}
+              >
+                +Add
+              </a>
+            )}
+            {editProject && projectId && (
+              <a
+                role="button"
+                className="ml-2 cursor-pointer"
+                onClick={(e) => {
+                  e.preventDefault();
+                  editProject();
+                }}
+              >
+                <GBEdit />
+              </a>
+            )}
+          </div>
+        ) : null}
         <div className="col-auto">
-          Project:{" "}
-          {projectIsDeReferenced ? (
-            <Tooltip
-              body={
-                <>
-                  Project <code>{projectId}</code> not found
-                </>
-              }
-            >
-              <span className="text-danger">
-                <FaExclamationTriangle /> Invalid project
-              </span>
-            </Tooltip>
-          ) : currentProject && currentProject !== experiment.project ? (
-            <Tooltip
-              body={<>This experiment is not in your current project.</>}
-            >
-              {projectId ? (
-                <strong>{projectName}</strong>
-              ) : (
-                <em className="text-muted">None</em>
-              )}{" "}
-              <FaExclamationTriangle className="text-warning" />
-            </Tooltip>
-          ) : projectId ? (
-            <strong>{projectName}</strong>
+          <Text weight="medium">Experiment Key: </Text>
+          {trackingKey ? (
+            <span>{trackingKey}</span>
+          ) : (
+            <em className="text-muted">None</em>
+          )}{" "}
+        </div>
+        <div className="col-auto">
+          <Text weight="medium">Created: </Text>
+          <span>{createdDate}</span>{" "}
+        </div>
+        <div className="col-auto">
+          <Text weight="medium">Owner: </Text>
+          {ownerName ? (
+            <span>
+              <InitialAvatar name={ownerName} size="sm" variant="soft" />{" "}
+              {ownerName}
+            </span>
           ) : (
             <em className="text-muted">None</em>
+          )}{" "}
+        </div>
+      </div>
+      <div className="row">
+        <div className="col-auto">
+          <Text weight="medium">Tags: </Text>
+          {experiment.tags?.length > 0 && (
+            <SortedTags
+              tags={experiment.tags}
+              skipFirstMargin={true}
```

Comment:
thinking it might be better to have no margins at all (useFlex) and instead use flex gap. Allows for more graceful wrapping.